internal/operatorcli/operator_cli.go

```diff
@@ -131,9 +151,14 @@ func RunRegister(c *cli.Context) error {
 		return cli.Exit(fmt.Sprintf("error creating logger %v", err), 1)
 	}
 
-	if passphrase == "" || keyStorePath == "" {
+	if passphrase != "" {
+		blsPassphrase = passphrase
+		ecdsaPassphrase = passphrase
+	}
```

Comment:
since there is default value to passphrase this will be true.
Need to reverse the order (or check if the new flags are empty)